BitwardenShared/UI/Auth/AuthCoordinator.swift

```diff
@@ -204,8 +204,10 @@ final class AuthCoordinator: NSObject, // swiftlint:disable:this type_body_lengt
             showMasterPasswordHint(for: username)
         case .preventAccountLock:
             showPreventAccountLock()
-        case let .removeMasterPassword(organizationName):
-            showRemoveMasterPassword(organizationName: organizationName)
+        case let .removeMasterPassword(organizationName: organizationName,
+                                       organizationId: organizationId,
+                                       keyConnectorUrl: keyConnectorUrl):
+            showRemoveMasterPassword(organizationName: organizationName, organizationId: organizationId, keyConnectorUrl: keyConnectorUrl)
```

Comment:
⛏️ Looks like this line is too long.

```suggestion
        case let .removeMasterPassword(
            organizationName: organizationName,
            organizationId: organizationId,
            keyConnectorUrl: keyConnectorUrl
        ):
            showRemoveMasterPassword(
                organizationName: organizationName,
                organizationId: organizationId,
                keyConnectorUrl: keyConnectorUrl
            )
```